src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,22 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion(val ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor : Vendedor) {
+        if(!vendedores.contains(vendedor)) {
+            vendedores.add(vendedor)
+        }
+        else {
+            Error()
+        }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:

```suggestion
        if(vendedores.contains(vendedor)) {
           error("Este vendedor ya está asociado al centro")
        }
        
        vendedores.add(vendedor)
```